### PR TITLE
configuration for avoid breaking the build if coverage is low

### PR DESCRIPTION
**Pull request by caarlos0**
- tests

### CODE REVIEW

File: src/main/java/com/contaazul/coverage/pullrequest/AbstractPullRequestValidator.java [hunk 1]
```diff
@@ -100,15 +103,14 @@ private Cobertura map(List<Cobertura> coverages) {
 	}
 
 	private void checkTotalCoverage(Cobertura cobertura) {
-		if (cobertura.isLowerThan( minCoverage ))
+		if (breakOnLowCoverage() && cobertura.isLowerThan( minCoverage ))
```
Comment (caarlos0) [inline src/main/java/com/contaazul/coverage/pullrequest/AbstractPullRequestValidator.java:106]: This chunk has 75.00% of tests coverage, which is lower than 90% minimum allowed

**Comment by caarlos0** [comment]
-1. Please add more tests!

**Comment by velo** [comment]
Lol

**Comment by caarlos0** [comment]
LOL

**Pull request merged**